src/module-init/module-init.c

```diff
@@ -115,6 +120,31 @@ void RS_moduleInfoFunc(RedisModuleInfoCtx *ctx, int for_crash_report) {
   // Fields statistics
   FieldsGlobalStats_AddToInfo(ctx);
 
+  // Memory
+  RedisModule_InfoAddSection(ctx, "memory");
+  TotalSpecsInfo total_info = RediSearch_TotalInfo();
+  RedisModule_InfoAddFieldDouble(ctx, "used_memory_indexes", total_info.total_mem);
+  RedisModule_InfoAddFieldDouble(ctx, "used_memory_indexes_human", total_info.total_mem / (float)0x100000);
+  RedisModule_InfoAddFieldDouble(ctx, "total_indexing_time", total_info.indexing_time / (float)CLOCKS_PER_MILLISEC);
+
+  // cursors
+  RedisModule_InfoAddSection(ctx, "cursors");
+  CursorList_Lock(&g_CursorsList);
+  CursorList_Lock(&g_CursorsListCoord);
+  RedisModule_InfoAddFieldLongLong(ctx, "global_idle",
+    ARRAY_GETSIZE_AS(&g_CursorsList.idle, Cursor **) + ARRAY_GETSIZE_AS(&g_CursorsListCoord.idle, Cursor **));
+  RedisModule_InfoAddFieldLongLong(ctx, "global_total",
+    kh_size(g_CursorsList.lookup) + kh_size(g_CursorsListCoord.lookup));
+  CursorList_Unlock(&g_CursorsListCoord);
+  CursorList_Unlock(&g_CursorsList);
```

Comment:
wouldn't it be better to have some cursor API call that will return/set in pointer the sizes and will handle the locks in `cursor,c`?

Reply:
Would be a bit better
Will add